src/src/Entity/User.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\UserRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+#[ORM\Entity(repositoryClass: UserRepository::class)]
+/**
+ * @method string getUserIdentifier()
+ */
+class User implements UserInterface, PasswordAuthenticatedUserInterface
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+    #[ORM\Column(length: 255)]
+    private ?string $name = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $first_name = null;
+
+    #[ORM\Column(length: 255, nullable: true)]
+    private ?string $email = null;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE, nullable: true)]
+    private ?\DateTimeInterface $createDate = null;
+
+    #[ORM\OneToMany(mappedBy: 'user', targetEntity: question::class, orphanRemoval: true)]
+    private Collection $question;
+
+    #[ORM\OneToMany(mappedBy: 'user', targetEntity: answer::class)]
+    private Collection $answer;
```

Comment:
Delete if possible to delete Answer and Question entities to have a clean PR ;)

Reply:
ok done